packages/icons/scripts/build.ts

```diff
@@ -24,4 +24,5 @@ async function build () {
   await fs.remove(iconsetPath)
 }
 
+// tslint:disable:no-floating-promises
 build()
```

Comment:
We could do something like this for better error handling here (and to get rid of the tslint:disable)

```suggestion
build().then(() => {
  console.log('icons successfully built')
}).catch((error) => {
  console.log('icons failed to build', error)
})
```